src/js/utils/themeDocUtils.js

```diff
@@ -84,9 +84,19 @@ export const themeDocUtils = {
   },
 
   inputStyle: {
-    'global.input.weight': {
-      description: 'The font weight of the text entered.',
-      type: 'number',
+    'global.input.font.height': {
```

Comment:
we should still keep `global.input.weight` and document how it is being replaced.